staff/bruno-diaz/arrays/concat.js

```diff
@@ -0,0 +1,49 @@
+function concat() {
```

Comment:
should `concat` have the minimum parameter `array`, that is, the target array you will concat other arrays with (the next arguments)